server/audit_math/macro.py

```diff
@@ -240,6 +248,10 @@ def compute_risk(
     """
     assert risk_limit < 1, "The risk-limit must be less than one!"
 
+    # We've done a full hand recount
+    if len(sample_results) == len(reported_results):
+        return 0, True
```

Comment:
we don't get a p-value in a full hand recount? what if the recount results don't match the reported results?

Reply:
So in a recount p-values no longer mean much. If you've counted all of your ballots by hand, then it is necessarily the case that there is zero chance that a computer malfunction could have caused a miscount (since no computers were used to do the count). While we could compute a p-value, it would be a little like estimating the chance it is raining while it is actively raining.

Reply:
might be nice to include that explanation as a comment for future readers